src/dex/balancer-v2/balancer-v2-integration.test.ts

```diff
@@ -397,3 +358,82 @@ describe('BalancerV2', function () {
     });
   });
 });
+
+describe('BeetsFi', () => {
+  it('FTM -> BOO: getPoolIdentifiers and getPricesVolume', async () => {
+    const dexKey = 'BeetsFi';
+    const network = Network.FANTOM;
+    const srcToken = Tokens[network]['FTM'];
+    const destToken = Tokens[network]['BOO'];
+    const amounts = [0n, BI_POWS[18]];
+
+    const dexHelper = new DummyDexHelper(network);
+    const blockNumber = await dexHelper.web3Provider.eth.getBlockNumber();
+    const beetsFi = new BalancerV2(network, dexKey, dexHelper);
+
+    await beetsFi.initializePricing(blockNumber);
+
+    const pools = await beetsFi.getPoolIdentifiers(
+      srcToken,
+      destToken,
+      SwapSide.SELL,
+      blockNumber,
+    );
+    console.log('Pool Identifiers: ', pools);
+
+    expect(pools.length).toBeGreaterThan(0);
+
+    const poolPrices = await beetsFi.getPricesVolume(
+      srcToken,
+      destToken,
+      amounts,
+      SwapSide.SELL,
+      blockNumber,
+      pools,
+    );
+    console.log('Pool Prices: ', poolPrices);
+
+    expect(poolPrices).not.toBeNull();
+    checkPoolPrices(poolPrices!, amounts, SwapSide.SELL, dexKey);
+
+    await beetsFi.releaseResources();
+  });
+  it('WETH -> FTM: getPoolIdentifiers and getPricesVolume', async () => {
+    const dexKey = 'BeetsFi';
+    const network = Network.FANTOM;
+    const srcToken = Tokens[network]['WETH'];
+    const destToken = Tokens[network]['FTM'];
+    const amounts = [0n, BI_POWS[18]];
+
+    const dexHelper = new DummyDexHelper(network);
+    const blockNumber = await dexHelper.web3Provider.eth.getBlockNumber();
+    const beetsFi = new BalancerV2(network, dexKey, dexHelper);
+
+    await beetsFi.initializePricing(blockNumber);
+
+    const pools = await beetsFi.getPoolIdentifiers(
+      srcToken,
+      destToken,
+      SwapSide.SELL,
+      blockNumber,
+    );
+    console.log('Pool Identifiers: ', pools);
+
+    expect(pools.length).toBeGreaterThan(0);
+
+    const poolPrices = await beetsFi.getPricesVolume(
+      srcToken,
+      destToken,
+      amounts,
+      SwapSide.SELL,
+      blockNumber,
+      pools,
+    );
+    console.log('Pool Prices: ', poolPrices);
```

Comment:
@Verisana Are you sure we need this `console.log` here?

Reply:
Not really. Personally, I don't see much usefulness in many of this `console.logs()` because I more tend to debug inside code rather than printing values, but for some of our collegues, it was useful, so we kept. We can remove, if we see that we are not using that. But I prefer it to do as separate task of improving our tests